test/unit/controllers/SignOutControllerSpec.scala

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions.AuthenticatedAction
+import models.SignOutReason
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, verify, verifyZeroInteractions, when}
+import org.scalatest.concurrent.ScalaFutures
+import play.api.test.Helpers._
+import play.twirl.api.HtmlFormat
+import views.html.{session_timed_out, user_signed_out}
+
+import scala.concurrent.ExecutionContext.global
+
+class SignOutControllerSpec extends ControllerLayerSpec with ScalaFutures {
+
+  private val authAction = mock[AuthenticatedAction]
+  private val mcc = stubMessagesControllerComponents()
+  private val sessionTimedOutPage = mock[session_timed_out]
+  private val userSignedOutPage = mock[user_signed_out]
+
+  private val controller = new SignOutController(SuccessfulAuth(), mcc, sessionTimedOutPage, userSignedOutPage)(global)
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+
+    when(sessionTimedOutPage.apply()(any(), any())).thenReturn(HtmlFormat.empty)
+    when(userSignedOutPage.apply()(any(), any())).thenReturn(HtmlFormat.empty)
+  }
+
+  override def afterEach(): Unit = {
+    reset(authAction)
+    reset(sessionTimedOutPage)
+    reset(userSignedOutPage)
```

Comment:
you can reset all services in one reset
`reset(authAction, sessionTimedOutPage, userSignedOutPage)`